client/src/dataset/components/HDF5ParamsForm.tsx

```diff
@@ -25,14 +26,16 @@ const HDF5ParamsForm: React.FC<MergedProps> = ({
     handleReset,
     onCancel,
     setFieldValue,
+    setFieldTouched,
 }) => {
     const dsItemsByPath: {
         [k: string]: DatasetInfoHDF5Item
     } = {};
     info?.datasets?.forEach(dsItem => dsItemsByPath[dsItem.path] = dsItem);
 
     const dsPathOptions = info?.datasets?.map(dsItem => {
-        const shape = dsItem.shape.join(",")
+        const raw_nav_shape = dsItem.raw_nav_shape.join(", ")
```

Comment:
Usually in TypeScript, the convention is to use `camelCase` for variable names, except of course if you are talking to the Python API.

Reply:
Good spot, I've adjusted a few more names too. What's worse is that I mixed the two in my additions!